adr/ADR-6.md

```diff
@@ -56,6 +58,6 @@ durable-name       = term
 js-internal-prefix = dollar (prefix dot)+
 js-user-prefix     = (prefix dot)+
 account-name       = (filename-safe)+ maximum 255 characters
+kv-key-name        = limited-term (dot limited-term)*
```

Comment:
```-/_=.a-zA-Z0-9```

Reply:
right backslash was escape, will fix

Reply:
they also may not start or end with `.`

```
Valid keys are \A[-/_=\.a-zA-Z0-9]+\z, additionally they may not start or end in ., after encoding
```

Please review ADR-8